x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -25,16 +38,13 @@ const App = () => {
           const route = routes[path];
           const Wrapper = () => {
             const { core } = usePluginContext();
+
+            const breadcrumb = [observabilityLabelBreadcrumb, ...route.breadcrumb];
             useEffect(() => {
-              core.chrome.setBreadcrumbs([
-                {
-                  text: i18n.translate('xpack.observability.observability.breadcrumb.', {
-                    defaultMessage: 'Observability',
-                  }),
-                },
-                ...route.breadcrumb,
-              ]);
-            }, [core]);
+              core.chrome.setBreadcrumbs(breadcrumb);
+            }, [core, breadcrumb]);
+
+            document.title = getTitleFromBreadCrumbs(breadcrumb);
```

Comment:
This should go inside of the `useEffect`. If it's out here in the render, we're calling it every time the component renders, like if you change the date range.